src/app/frontend/logs/logstoolbar/logstoolbar.scss

```diff
@@ -0,0 +1,23 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.md-logs-color-icon {
+  background-color: rgba(0, 0, 0, 1);
+  color: rgba(255, 255, 255, 1);
+}
+
+.md-logs-color-icon-invert {
+  background-color: rgba(255, 255, 255, 1);
+  color: rgba(0, 0, 0, 1) !important;
```

Comment:
`!important`? Avoid it whenever possible. Why is it needed here? Cannot you rely on rule order or specificity?